File: Layer_datas/Services/Objects/Log.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Layer_datas.Services.Objects
{
    public class Log
    {
        public Int64 id { get; set; }
        public DateTime date { get; set; }
        public string level { get; set; }
        public string message { get; set; }
        public string detail { get; set; }
        public users.User user { get; set; }
    }
}

File: Swiman/__scripts__/modules/api/models/Log.ts


    
        import { User } from "./User";
    

    import * as assign from "object-assign";

    

    


    class Log {
        

        
        // ID
        public id: number = 0;
        // DATE
        public date: Date = undefined;
        // LEVEL
        public level: string = undefined;
        // MESSAGE
        public message: string = undefined;
        // DETAIL
        public detail: string = undefined;
        // USER
        public user: User = undefined;

        constructor(init?: Partial<Log>){
            assign(this, init);
        }
        
    }


export {  Log }



